CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ Markdown Spec](https://github.github.com/gfm/).
 - Typo in CentOS RAIS docs for SELinux file context application command
 - Mistake in path for local settings copy command and outdated, non-general
   examples in CentOS OpenONI web app configuration documentation
+- Tests are properly isolated from the local environment
```

Comment:
I think it'd be more informative to point out specifically that the Python virtual environment is the part now isolated that wasn't before :)

Reply:
Done!